src/app/portlets/content-types/form/content-types-form.component.html

```diff
@@ -26,55 +26,55 @@
         <div class="ui-g">
             <div class="ui-g-6">
                 <div class="form__group" *ngIf="isEditMode">
-                    <b>{{i18nMessages['Variable']}}:</b>
+                    <b>{{i18nMessages['contenttypes.variable']}}:</b>
                     <a href="#">{{data.variable }}</a>
                 </div>
                 <div class="form__group">
-                    <label for="content-type-form-description" class="form__label">{{i18nMessages['description']}}</label>
+                    <label for="content-type-form-description" class="form__label">{{i18nMessages['contenttypes.form.label.description']}}</label>
                     <input pInputText type="text" id="content-type-form-description" name="description" formControlName="description" />
                 </div>
                 <div class="form__group">
-                    <label for="content-type-form-host" class="form__label">{{i18nMessages['Host-Folder']}}</label>
+                    <label for="content-type-form-host" class="form__label">{{i18nMessages['contenttypes.form.field.host.folder']}}</label>
```

Comment:
I not sure about the field word here
Maybe this culd be in this way 'contenttypes.form.field.host_folder' because host_folder in one content types 's property, this way  '...host.folder' it look like we have host and into it we have a property o something call folder.
Also I would add the word label in the end: contenttypes.form.field.host_folder.label

Reply:
Changes were added in the latest commit, thanks!